Makefile

```
@@ -81,34 +87,36 @@ oc-all-clean:
 	test -n "$(APP_NAME)" # Please provide an app name via APP_NAME=openshift-launchpad
 	@echo "+\n++ Tearing down all OpenShift objects created from templates...\n+"
 	@oc project $(NAMESPACE)
-	@oc delete all -l app=$(APP_NAME)
+	@oc delete all -l app=$(APP_NAME) --ignore-not-found
```

`--ignore-not-found` not needed when filtering by label
```suggestion
	@oc delete all -l app=$(APP_NAME)
```